src/main/java/com/twilio/http/Request.java

```diff
@@ -266,6 +270,8 @@ public void addPostParam(final String name, final String value) {
      * @param value value of parameter
      */
     public void addHeaderParam(final String name, final String value) {
+        if (value == null)
+            return;
```

Comment:
Could this be moved to the `addParam` method instead of repeated?

Reply:
Yes thought about this but confused about the reason to create these wrapper function over addParams, so I added the null checks in wrapper but agree it's breaking DRY principle. Anyways made the changes